Revive/main.cpp

```diff
@@ -35,24 +36,40 @@ HANDLE WINAPI HookOpenEvent(DWORD dwDesiredAccess, BOOL bInheritHandle, LPCWSTR
 	return TrueOpenEvent(dwDesiredAccess, bInheritHandle, lpName);
 }
 
-HMODULE WINAPI HookLoadLibrary(LPCWSTR lpFileName)
+HMODULE WINAPI HookLoadLibraryW(LPCWSTR lpFileName)
 {
 	LPCWSTR name = PathFindFileNameW(lpFileName);
 	LPCWSTR ext = PathFindExtensionW(name);
 	size_t length = ext - name;
 
 	// Load our own library again so the ref count is incremented.
-	if (wcsncmp(name, ovrModuleName, length) == 0)
-		return TrueLoadLibrary(revModuleName);
+	if (wcsncmp(name, ovrModuleName, length) == 0) {
+		return TrueLoadLibraryW(revModuleName);
+	}
+		return TrueLoadLibraryW(lpFileName);
+
```

Comment:
Please follow the original code style by removing the brackets. Also please fix the indenting.

Reply:
Updated